ITISFoundation/osparc-simcore
# ✨🎨 Preferences: Inactivity and Concurrency

## What do these changes do?

* Adds two new preferences: The inactivity and the maximum job concurrency settings

![image](https://github.com/ITISFoundation/osparc-simcore/assets/4764217/b599ae95-97d1-495f-9473-3b620860b317)


## Related issue/s




## How to test

Check the preferences dialog for the mentioned properties and check that they change in the DB.
**Please review the wordings.**


## Dev Checklist


## DevOps Checklist
